data/pc/1.20.3/proto.yml

```diff
@@ -1104,8 +1104,11 @@
       block_interaction_type: varint
       small_explosion_particle: particle
       large_explosion_particle: particle
-      explosion_sound_name: string
-      sound_range?: f32
+      soundId: varint
+      _: soundId ?
```

Comment:
this is super weird; are you sure? 

it means sounds id must be 0 or the packet is invalid since there is no default

Reply:
going to add default: void

Reply:
No default defaults to void anyway no?

Reply:
no, no default means there is no default: other values are impossible and should crash